doc/releases/release-notes-3.4.rst

```diff
@@ -140,6 +140,9 @@ Changes in this release
   using picolibc, only applications using picolibc will be affected by this
   change at this time.
 
+* Changed implementation of checking for bounded endpoints during closing
+  an IPC instance. The new one checks by bounded flag.
```

Comment:
This is in the wrong section. Please add a new `IPC` section under the `Subsystems` section and reword like:
```
:c:func:`ipc_service_close_instance` now only acts on bounded endpoints.
```